Ziffity/AddCustomer/etc/module.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Ziffity_AddCustomer" setup_version="1.0.0">
```

Comment:
@msakila532 Magento 2.3.x was moved to the declarative scheme concept so there is no need for the stetup_version

Reply:
Removed